app/scripts/lib/app-start.js

```diff
@@ -507,15 +507,19 @@ function (
      * Sets a UUID value that is unrelated to any account information.
      * This value is useful to determine if the logged out user qualifies for ab testing or metrics.
      */
-    initializeUuid: function () {
+    initializeUniqueUserId: function () {
       var storage = Storage.factory('localStorage', this._window);
 
       // we reuse the same uuid if it is available in local storage.
+      //
+      // Note from stomlinson, I changed the internal name to `uniqueUserId`
+      // from `uuid`, but kept `uuid` in localStorage so that a localStorage
+      // migration was not needed.
       if (storage.get('uuid')) {
-        this._uuid = storage.get('uuid');
+        this._uniqueUserId = storage.get('uuid');
       } else {
-        this._uuid = uuid.v4();
-        storage.set('uuid', this._uuid);
+        this._uniqueUserId = uuid.v4();
+        storage.set('uuid', this._uniqueUserId);
```

Comment:
FWIW, work that @philbooth is doing to fix https://github.com/mozilla/fxa-content-server/issues/2480 will affect this code.